internal/venice-common/src/main/java/com/linkedin/venice/ConfigKeys.java

```diff
@@ -2456,10 +2456,25 @@ private ConfigKeys() {
    */
   public static final String CONTROLLER_HELIX_RESOURCE_CAPACITY_WEIGHT = "controller.helix.default.instance.capacity";
 
+  /**
+   * Specifies how frequently the deferred version swap service runs. Default value is 1 minute
+   */
   public static final String CONTROLLER_DEFERRED_VERSION_SWAP_SLEEP_MS = "controller.deferred.version.swap.sleep.ms";
+
+  /**
+   * Enables / disables the deferred version swap service. Default value is disabled
+   */
   public static final String CONTROLLER_DEFERRED_VERSION_SWAP_SERVICE_ENABLED =
       "controller.deferred.version.swap.service.enabled";
 
+  /**
+   * Enables / disables allowing dvc clients to perform a target region push with deferred swap. When enabled, dvc clients
```

Comment:
So this is temporary till we support delayed DVC ingestion right? can you mention that